opm/simulators/linalg/bda/openclSolverBackend.cpp

```diff
@@ -663,7 +663,18 @@ void openclSolverBackend<block_size>::solve_system(WellContributions& wellContri
     Timer t;
 
     // actually solve
-    gpu_pbicgstab(wellContribs, res);
+    try {
+        gpu_pbicgstab(wellContribs, res);
+    } catch (cl::Error error) {
+        std::ostringstream oss;
+        oss << "openclSolverBackend::solve_system error: " << error.what() << "(" << error.err() << ")\n";
```

Comment:
What does this print? As a codeveloper I would like to see the constant/enum name rather than a numeric value as I otherwise need to check the opencl headers.

Reply:
It prints both some default error description (`clBuildProgram`) and the error ID (-11).
The next line prints the error code translated using the values in `cl.h` (`CL_BUILD_PROGRAM_FAILURE`) along with any extra info that was added to `opencl.cpp`.